load-test-framework/src/main/java/com/google/pubsub/flic/processing/MessageProcessingHandler.java

```diff
@@ -140,14 +140,16 @@ public void printStats(long start, DelayTrackingThreadPool executor, AtomicBoole
                 latencyStats.getValueAtPercentile(50),
                 latencyStats.getValueAtPercentile(95),
                 latencyStats.getValueAtPercentile(99)));
-    log.info(
-        "The average delay for processing "
-            + latencyType
-            + " latency was "
-            + executor.getAverageTaskWaitTime()
-            + ""
-            + " "
-            + "ms");
+    if(executor != null) {
```

Comment:
Should executor ever be null? You should use an annotation above for @Nullable if so.